dashboard/app/controllers/home_controller.rb

```diff
@@ -167,9 +167,11 @@ def terms_and_privacy
 
       unless current_user.donor_teacher_banner_dismissed
         donor_banner_name = current_user.school_donor_name
+        afe_eligible = current_user.school_info.school.afe_high_needs?
       end
 
       donor_banner_name ||= params[:forceDonorTeacherBanner]
+      afe_eligible ||= params[:forceDonorTeacherBanner]
```

Comment:
Where is the `forceDonorTeacherBanner` param getting set? I don't see it in codebase –– could we delete this too?

Reply:
It allows you to force the banner to show using `?forceDonorTeacherBanner=1`. I could see going either way but it can be helpful to have the ability to force to show it